src/core/ext/filters/client_channel/resolver/dns/event_engine/event_engine_client_channel_resolver.cc

```diff
@@ -475,12 +475,8 @@ absl::optional<Resolver::Result> EventEngineClientChannelDNSResolver::
     return std::move(result);
   }
   if (!errors_.ok()) {
-    result.resolution_note =
-        std::string(errors_
-                        .status(absl::StrCat("errors resolving ",
-                                             resolver_->name_to_resolve()),
-                                absl::StatusCode::kUnavailable)
-                        .message());
+    result.resolution_note = std::string(errors_.message(
```

Comment:
Don't think we need the explicit conversion to `std::string` anymore, since `errors_.message()` already returns that type.

Reply:
Thanks, fixed.